relays/tasmota/manifest.yml

```diff
@@ -0,0 +1,85 @@
+# Manifest for blueprint connection to any Tasmota device
+
+blueprint_spec: device/1.0
+
+display_name: Tasmota device
+description: A wrapper around the Tasmota interface to connect to any Tasmota powered device.
+icon: light-switch
+vendor: Tasmota
+author: Hermanos Energy
+support:
+  url: https://go.enapter.com/enapter-blueprint-support
+  email: support@enapter.com
+contributors:
+  - Jan Ruijtenberg
+license: MIT
+
+
+communication_module:
+  product: ENP-VIRTUAL
+  lua:
+    file: firmware.lua
+    dependencies:
+      - enapter-ucm
+
+properties:
+  serial_number:
+    type: string
+    display_name: Serial Number
+  model:
+    type: string
+    display_name: Model
+
+telemetry:
```

Comment:
I'd suggest to add `status` (https://developers.enapter.com/docs/reference/manifest#device-status), it will look a bit nicer in UI:

```suggestion
telemetry:
  status:
    display_name: Status
    type: string
    enum:
      - "on"
      - "off"
```

Reply:
Is there a way to also populate the wifi, and mac settings in the UI, they are currently empty but I could get them from the tasmota status fields. But dont know how to populate them 

![afbeelding](https://github.com/user-attachments/assets/c952afeb-95ff-4ede-8371-bedf7230ab0b)

Reply:
Good point. Those fields are actually reserved for UCM WiFi/IP. Maybe we need to hide them from this list for Virtual UCMs.

What you can do is to add and send properties `wifi_ssid` etc, they will be shown below this block with the same UI.